totalRP3/Core/Globals.lua

```diff
@@ -141,4 +141,5 @@ TRP3_API.globals.build = function()
 	end
 end
 
-TRP3_API.globals.addon = LibStub("AceAddon-3.0"):NewAddon(TRP3_API.globals.addon_name);
+TRP3_Addon = LibStub("AceAddon-3.0"):NewAddon("TRP3");
```

Comment:
Assuming this won't be an issue but `addon_name = "Total RP 3"` so it changes the behaviour

Reply:
I'd be surprised if it breaks anything else - the addon object is only really used for our bootstrap logic. I just updated it so it looks a bit more sensible in `/dump` output, since Ace concats submodule names to it via an underscore and "Total RP 3_Foo" looks odd.